### PR TITLE
Add more padding to sidebar to better support multiple headings

### PR DESCRIPTION
**Pull request by jordanmaguire**
screen shot 2016-03-10 at 3 55 03 pm

### CODE REVIEW

File: app/assets/stylesheets/admin/modules/_sidebar.sass [hunk 1]
```diff
@@ -12,10 +12,12 @@ aside[role="complementary"]
     font-size: 1.3em
     margin: 0
 
+  h2
+    margin: $small-spacing 0px
```
Comment (pzi) [inline app/assets/stylesheets/admin/modules/_sidebar.sass]: Try `margin: $small-spacing 0 0`?

Comment (lordmortis) [inline app/assets/stylesheets/admin/modules/_sidebar.sass]: agreed

**Comment by lordmortis** [comment]
seems good other than the comments :+1:

**Comment by jordanmaguire** [comment]
Adjusted margins :+1:

**Pull request merged**